lib/_stream_readable.js

```diff
@@ -522,8 +524,13 @@ Readable.prototype.pipe = function(dest, pipeOpts) {
     if (false === ret) {
       debug('false write response, pause',
             src._readableState.awaitDrain);
```

Comment:
Would it be better to split it up into "false write response" where it is now and something like "stream pausing" inside the next `if`?